src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1875,11 +1875,12 @@ public OpcodePushDelegateRelocateLater(string destLabel, bool withClosure) : bas
         protected OpcodePushDelegateRelocateLater() : base() {}
         
         public override void PopulateFromMLFields(List<object> fields)
-        {
+        {            
             // Expect fields in the same order as the [MLField] properties of this class:
             if (fields == null || fields.Count<1)
                 throw new Exception("Saved field in ML file for OpcodePushDelegateRelocatelater seems to be missing.  Version mismatch?");
-            WithClosure = (bool)fields[0];
+            DestinationLabel = (string)( fields[0] ); // this is really from the base class.
+            WithClosure = (bool)fields[1];
```

Comment:
I think the original reason this was a derived class is that there's a few places where I try to catch both kinds of relocatable opcode of them with an 'is' check.  (i.e. catch both the base and the derived opcode in the same check)

That may be changed to perhaps an Opcode base class field for things like "isRelocatable", and then what you say could work.